Packs/VMware/Integrations/VMware/VMware.py

```diff
@@ -373,48 +554,310 @@ def change_nic_state(args):
+        data.append({
+            'TagName': args.get('tag'),
+            'Category': args.get('category'),
+            'VM': vm.name
+        })
+    ec = {
+        'VMWareTag(val.TagName && val.Category && val.TagName == obj.TagName && va.Category == obj.Category)': data
+    }
+    return {
+        'ContentsFormat': formats['json'],
+        'Type': entryTypes['note'],
+        'Contents': data,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('Virtual Machines with Tag {}'.format(args.get('tag')), data, removeNull=True),
+        'EntryContext': ec
+    }
+
+
+def create_vm(si, args):
+    content = si.RetrieveContent()
+    folder = search_for_obj(content, [vim.Folder], args.get('folder'))
+    host = search_for_obj(content, [vim.HostSystem], args.get('host'))
+    if not host:
+        raise Exception('The host provided is not valid.')
+    pool = host.parent.resourcePool
+    spec = create_vm_config_creator(host, args)
+
+    task = folder.CreateVM_Task(config=spec, pool=pool, host=host)
+    wait_for_tasks(si, [task])
+
+    if task.info.state == 'success':
+        mac_address = ''
+        summary = task.info.result.summary
+
+        try:
+            for dev in task.info.result.config.hardware.device:
+                if isinstance(dev, vim.vm.device.VirtualEthernetCard):  # type: ignore
+                    mac_address = dev.macAddress
+                    break
+        except Exception:  # noqa
+            pass
+
+        data = {
+            'Name': summary.config.name,
+            'Template': summary.config.template,
+            'Path': summary.config.vmPathName,
+            'Guest': summary.config.guestFullName,
+            'UUID': summary.config.instanceUuid,
+            'IP': summary.guest.ipAddress if summary.guest.ipAddress else '',
+            'State': summary.runtime.powerState,
+            'HostName': summary.guest.hostName if summary.guest.hostName else '',
+            'MACAddress': mac_address,
+            'Snapshot': task.info.result.snapshot.currentSnapshot if task.info.result.snapshot else '',
+            'SnapshotCreateDate': '',
+            'SnapshotUUID': '',
+            'Deleted': False
+        }
+        ec = {
+            'VMWare(val.UUID && val.UUID === obj.UUID)': data
+        }
+        return {
+            'ContentsFormat': formats['json'],
+            'Type': entryTypes['note'],
+            'Contents': data,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('Virtual Machine', data,
+                                             headers=['Name', 'Template', 'Path', 'Guest', 'UUID',
+                                                      'IP', 'State', 'HostName', 'MACAddress', 'SnapshotCreateDate',
+                                                      'SnapshotUUID', 'Deleted'], removeNull=True),
+            'EntryContext': ec
+        }
+    elif task.info.state == 'error':
```

Comment:
relevant for other commands:
1. state is always binary? success/error?
2. can we add debug logs with additional data?